src/components/Login/JellyfinLogin.tsx

```diff
@@ -114,8 +115,9 @@ const JellyfinLogin: React.FC<JellyfinLoginProps> = ({
             //   throw new Error('Invalid serverType'); // You can customize the error message
             // }
 
-            const res = await fetch('/api/v1/auth/jellyfin', {
+            const res = await apiFetch('/api/v1/auth/jellyfin', {
               method: 'POST',
+              credentials: 'same-origin',
```

Comment:
You forgot to remove it (still not an issue ofc)

Reply:
Let me remove it